admin/BundleSearchIndexesDump

```diff
@@ -189,7 +193,7 @@ for my $hash_program ('md5sum', 'sha256sum') {
 _info 'Successfully bundled SolrCloud backups as search indexes dump.';
 exit;
 
-################################################################################
+##########                          '-.((*_.-'                        ##########
```

Comment:
Am I supposed to know what these hieroglyphs do? 😆"Make separating comments more useful" doesn't explain anything.

Reply:
LOL! I've decoded the Easter surprise, well played. 😆